x/accounts/keeper_test.go

```
@@ -3,7 +3,9 @@ package accounts
 import (
 	"testing"
 
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/cosmos/gogoproto/types"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cosmos/gogoproto/types" is being imported more than once

(stylecheck)